app/Commands/Session/EndCommand.php

```
@@ -0,0 +1,174 @@
+        }
+    }
+
+    private function detectProject(): string
+    {
+        $cwd = getcwd();
+        if ($cwd === false) {
+            return 'unknown';
+        }
+
+        $gitRoot = shell_exec('git rev-parse --show-toplevel 2>/dev/null');
+        if ($gitRoot !== null && trim($gitRoot) !== '') {
```

PHPStan (2.1.33)

Parameter #1 $string of function trim expects string, string|false given.


(argument.type)